WeakAuras/DiscordList.lua

```diff
@@ -27,18 +31,21 @@ Private.DiscordList = {
   [=[Listefano]=],
   [=[Luckyone]=],
   [=[Luxthos]=],
+  [=[m33shoq]=],
   [=[MetalMusicMan]=],
+  [=[NostraDumAzz]=],
   [=[Oi]=],
   [=[Ora]=],
   [=[phoenix7700]=],
   [=[Pseiko]=],
   [=[Reloe]=],
+  [=[Shwinsta]=],
   [=[Spaten]=],
-  [=[Steedcarrell]=],
   [=[update]=],
   [=[vozochris]=],
   [=[Wizeowel]=],
   [=[Xepheris]=],
+  [=[ᴊɪʙᴇʀɪsʜ]=],
```

Comment:
will this show correctly?

Reply:
nop
![image](https://github.com/user-attachments/assets/0dccc996-07e3-46ec-81ff-03a82947a04f)

Reply:
Yeah I'll take a look at improving the filtering to check what didn't work there.

Reply:
And improve the reporting on who added which name too, since I think that's not visible atm.

Reply:
I also sent them a message to please change their name.